yarn-error.log

```diff
@@ -0,0 +1,32 @@
+Arguments: 
```

Comment:
this should be in .gitignore 

now we all know that you use Windows :)

Reply:
lmao windows is for squares jk lol